pandas/errors/__init__.py

```diff
@@ -38,9 +38,53 @@ class ParserError(ValueError):
 
 class DtypeWarning(Warning):
     """
-    Warning that is raised for a dtype incompatibility. This
-    can happen whenever `pd.read_csv` encounters non-
-    uniform dtypes in a column(s) of a given CSV file.
+    Warning raised when importing different dtypes in a column from a file.
+
+    Raised for a dtype incompatibility. This can happen whenever `pd.read_csv`
+    or `pd.read_table` encounter non-uniform dtypes in a column(s) of a given
+    CSV file.
+
+    It only happens when dealing with larger files.
```

Comment:
instead of this comment, you can add in the Notes section, that this can happen in larger files. Its because the dtype checking happens per chunk that is read.

Reply:
Tks!